src/library/trackmodel.h

```diff
@@ -99,6 +99,8 @@ class TrackModel {
     // Gets the track ID of the track at the given QModelIndex
     virtual TrackId getTrackId(const QModelIndex& index) const = 0;
 
+    virtual CoverInfo getCoverInfo(const QModelIndex& index) const { return CoverInfo(); }
```

Comment:
`= 0` to enforce override. You already forgot to implement this function for ProxyTrackModel.

Reply:
I implemented this function, but I'm unsure, if this is what you expect here. Could you have a look please.

Reply:
Yes, that's correct.

Reply:
Thanks for the quick response!